tests/test_logger.py

```
@@ -2,272 +2,287 @@
-        
+
         assert logger1 is logger2
-    
-    @patch.dict('os.environ', {'LOG_FILE_LEVEL': 'DEBUG'})
+
+    @patch.dict("os.environ", {"LOG_FILE_LEVEL": "DEBUG"})
     def test_logger_file_level_from_env(self):
         """Test setting file log level from environment variable"""
         # Reset singleton to pick up new environment
         Logger._instance = None
         logger = Logger()
```

Ruff (0.13.1)

Local variable `logger` is assigned to but never used

Remove assignment to unused variable `logger`

(F841)